core/python_utils.py

```diff
@@ -218,6 +219,23 @@ def open_file(filename, mode, encoding='utf-8', newline=None):
         raise IOError('Unable to open file: %s' % filename)
 
 
+def get_package_file_contents(package: str, filepath: str) -> str:
+    """Open file and return a corresponding file object.
+
+    Args:
+        package: str. The file to be opened.
+        filepath: str. The file to be opened.
+
+    Returns:
+        _io.TextIOWrapper. The file object.
```

Comment:
Is the _ supposed to be there? io seems public.

Reply:
No, changed to str.